sorting accented characters (#2063)
meilisearch/documentation: learn/advanced/sorting.md

2. Add those attributes to the `sortableAttributes` index setting
3. Update Meilisearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)

::: note

> Meilisearch sorts strings in lexicographic order based on their byte values. This means, `á` which has a value of 225, will be sorted after `z`, which has a value of 122.  

Reviewer: ```suggestion
Meilisearch sorts strings in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.
```

> Meilisearch converts documents to lowercase for sorting, they remain unchanged in the results. They are then sorted in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.

Author: That's a lot clearer. Thanks

3. Update Meilisearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)

::: note

> Documents used for sort are converted to lowercase and then sorted in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.

Reviewer: For me this wording raises the question of whether the fields in the returned documents will actually be lowercased. I'm guessing that they're only lowercased for the purposes of sorting and will be unchanged in the results. If so, could you rephrase to make that clear?
